releases/publish.go

```diff
@@ -160,28 +160,24 @@ func PublishPluginFeed(plugin string) {
 	publishPackageFeed("plugin", plugin)
 }
 
-func generatePackageFeed(pkgType string) {
+func generatePackageFeed(pkgType string) error {
 	pkgDir := pkgType + "s"
 	feedFile := filepath.Join(packagesRepo, pkgDir, "atom.xml")
-
-	// Try to use a local copy of porter first, otherwise use the
-	// one installed in GOAPTH/bin
-	porterPath := "bin/porter"
-	if _, err := os.Stat(porterPath); err != nil {
-		porterPath = "porter"
-		tools.EnsurePorter()
+	if err := os.MkdirAll(filepath.Dir(feedFile), 0770); err != nil {
+		return err
 	}
-	must.RunV(porterPath, "mixins", "feed", "generate", "-d", filepath.Join("bin", pkgDir), "-f", feedFile, "-t", "build/atom-template.xml")
+
+	return shx.RunE("porter", "mixins", "feed", "generate", "-d", filepath.Join("bin", pkgDir), "-f", feedFile, "-t", "build/atom-template.xml")
```

Comment:
This was changed to take advantage of porter being in the local bin, and to help make it more testable

Reply:
It works as-is with just "porter" because I am prepending `$PWD/bin` to the processes PATH environment variable. When I was adding tests, I was creating a tmp directory and installing porter into that. Hard coding bin/porter required more workarounds to make the test work, vs relying on resolving from PATH. But if this seems problematic I can update the tests to chdir into the temp directory and restore the pwd after each test is run.

Just keep in mind that even when setting it to bin/porter, we still need to ensure PORTER_HOME is overridden.

Suggestions welcome for how to make this less easy to mess up!